lib/hound/helpers/log.ex

```diff
@@ -7,6 +7,8 @@ defmodule Hound.Helpers.Log do
   Fetches console.log() from the browser as a single string of log-lines.
   """
   def fetch_log() do
+    fail_if_webdriver_selenium("fetch_log()")
```

Comment:
Logs doesn't work only for Firefox on selenium, so I'm not sure it's correct to fail on selenium webdriver, if there is a way to know that the browser is Firefox and then fail it will be much better